<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_total_pages">
     <property name="text">
      <string>Total Pages:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_from">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBox_to">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_to</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>to_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_from</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>from_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>to_changed(int)</slot>
  <slot>from_changed(int)</slot>
 </slots>
</ui>
